resources/qml/dialogs/UserProfile.qml

```diff
@@ -299,6 +312,25 @@ ApplicationWindow {
                     onClicked: profile.refreshDevices()
                 }
 
+                ImageButton {
```

Comment:
Not sure if this is the right place for that button. Maybe the room settings are better, even though ignores are global?

Reply:
Can you put this into the global settings next to where the button here is: https://github.com/Nheko-Reborn/nheko/pull/1537 ?